proguard.pro

```diff
@@ -11,8 +11,8 @@
 
 -keep,allowoptimization public class dev.nolij.zume.api.** { public *; } # public APIs
 -keepclassmembers class dev.nolij.zume.impl.config.ZumeConfigImpl { public <fields>; } # dont rename config fields
--keep,allowoptimization class dev.nolij.zume.ZumeMixinPlugin # dont rename mixin plugin
--keep @org.spongepowered.asm.mixin.Mixin class dev.nolij.zume.** { *; } # dont touch mixins
+-keep,allowoptimization,allowobfuscation class dev.nolij.zume.ZumeMixinPlugin
+-keep @org.spongepowered.asm.mixin.Mixin class ** { *; } # dont touch mixins
```

Comment:
This doesn't error? Last time I tried to add a `**` filter to ProGuard it caused errors...

Reply:
seems to work, both locally and in CI